server/test/services/question/types/QuestionDefinitionBuilderTest.java

```diff
@@ -93,4 +94,32 @@ public void builder_withEnumeratorQuestion_keepsEntityType() throws Exception {
     assertThat(enumerator.getEntityType().isEmpty()).isFalse();
     assertThat(enumerator.getEntityType().getDefault()).isEqualTo("household member");
   }
+
+  @Test
+  public void getLastModifiedTimeWhenExists() throws Exception {
+    Instant now = Instant.now();
+    QuestionDefinition questionDefinition =
+        new QuestionDefinitionBuilder()
+            .setQuestionType(QuestionType.TEXT)
+            .setName("")
+            .setDescription("")
+            .setQuestionText(LocalizedStrings.of())
+            .setQuestionHelpText(LocalizedStrings.empty())
+            .setLastModifiedTime(Optional.of(now))
+            .build();
+    assertThat(questionDefinition.getLastModifiedTime()).isPresent();
```

Comment:
You are not checking that it's equal to the provided time?

Reply:
That was a complete oversight. Thanks for noticing!